src/test/resources/com.uid2.shared/test/secure/azurecc/jwt_payload_aks.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
not about this file but rename jwt_payload.json to jwt_payload_aci.json as well?